### PR TITLE
[DONE] revamp scenario page

### PR DESCRIPTION
**Pull request by fritzvd**
## problem
scenarios was completely seperated from the interface. Which made it hard to use.

## solution
Embed scenarios as an omnibox feature.

Todo
- [x] add pagination (for loadtime and sanity)
- [x] add a scenario card for the omnibox with more detailed information
- [x] have the card and interface be available after toggling a result as a map preview

### CODE REVIEW

File: app/components/scenarios/scenarios.html [hunk 1]
```diff
@@ -1,52 +1,75 @@
...
-      </div>
+<div class="card" ng-controller="ScenariosCtrl">
+  <div class="card-content">
+    <div ng-if="loading">
+      <i class="fa fa-refresh fa-spin"></i>
+    </div>
+
+    <div ng-if="selectedScenario === null">
+      <span id="title-holder" class="card-title-text ng-binding" translate>
+        3di resultaten
```
Comment (ernstkui) [inline app/components/scenarios/scenarios.html]: Mooi dat er aan gedacht wordt om de string te annoteren met `translate`, maar nu staan er nederlandse string op transifex die naar het Nederlands vertaald moeten worden... :hankey: 

https://github.com/nens/lizard-client/blob/master/README.md#internationalization 

The app is in English which is translated to other languages.

Comment (fritzvd) [inline app/components/scenarios/scenarios.html]: :hocho: :broken_heart:

2015-10-01 11:44 GMT+02:00 Ernst Kuilder <notifications@github.com>:

> In app/components/scenarios/scenarios.html
> <https://github.com/nens/lizard-client/pull/451#discussion_r40895737>:
>
> > -            </tr>
> > -          </tbody>
> > -
> > -        </table>
> > -
> > -
> > -      </div>
> > +<div class="card" ng-controller="ScenariosCtrl">
> > +  <div class="card-content">
> > +    <div ng-if="loading">
> > +      <i class="fa fa-refresh fa-spin"></i>
> > +    </div>
> > +
> > +    <div ng-if="selectedScenario === null">
> > +      <span id="title-holder" class="card-title-text ng-binding" translate>
> > +        3di resultaten
>
> Mooi dat er aan gedacht wordt om de string te annoteren met translate,
> maar nu staan er nederlandse string op transifex die naar het Nederlands
> vertaald moeten worden... [image: :hankey:]
>
>
> https://github.com/nens/lizard-client/blob/master/README.md#internationalization
>
> The app is in English which is translated to other languages.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nens/lizard-client/pull/451/files#r40895737>.
>



-- 
*Nelen & Schuurmans *

Postbus 1219  3500 BE Utrecht
Zakkendragershof 34-44 Utrecht
tel. 030 2330 200
@fritzvd <http://twitter.com/fritzvd>
www.nelen-schuurmans.nl

Comment (ernstkui) [inline app/components/scenarios/scenarios.html]: Het is allemaal wat

Comment (fritzvd) [inline app/components/scenarios/scenarios.html]: https://www.youtube.com/watch?t=4&v=KrZHPOeOxQQ

On 1 October 2015 at 11:51, Ernst Kuilder <notifications@github.com> wrote:

> In app/components/scenarios/scenarios.html
> <https://github.com/nens/lizard-client/pull/451#discussion_r40896303>:
>
> > -            </tr>
> > -          </tbody>
> > -
> > -        </table>
> > -
> > -
> > -      </div>
> > +<div class="card" ng-controller="ScenariosCtrl">
> > +  <div class="card-content">
> > +    <div ng-if="loading">
> > +      <i class="fa fa-refresh fa-spin"></i>
> > +    </div>
> > +
> > +    <div ng-if="selectedScenario === null">
> > +      <span id="title-holder" class="card-title-text ng-binding" translate>
> > +        3di resultaten
>
> Het is allemaal wat
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nens/lizard-client/pull/451/files#r40896303>.
>



-- 
*Nelen & Schuurmans *

Postbus 1219  3500 BE Utrecht
Zakkendragershof 34-44 Utrecht
tel. 030 2330 200
@fritzvd <http://twitter.com/fritzvd>
www.nelen-schuurmans.nl

**Pull request merged**